docs/source/en/api/pipelines/stable_diffusion/pix2pix.mdx

```diff
@@ -46,7 +46,7 @@ from diffusers import StableDiffusionInstructPix2PixPipeline
 
 model_id = "timbrooks/instruct-pix2pix"
 pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(
-    model_id, torch_dtype=torch.float16, revision="fp16", safety_checker=None
+    model_id, torch_dtype=torch.float16, safety_checker=None
```

Comment:
`safety_checker` checker is still `None` here.

Reply:
was about to comment the same thing!

Reply:
Yeah let's open a new PR to fix it